package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headquarters-node",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
But the versioning refers to this package, and not the api it serves. There isn't any new features, and this change is only necessary to fix a bug introduced by a change on the api. And that fix required changing a single line of code without any breaking changes from the viewpoint of the people using this package. Reading [here](http://semver.org/) this fits right into a patch, and not a minor change.